File: sdk/src/main/java/com/robotemi/sdk/listeners/OnRobotReadyListener.java
package com.robotemi.sdk.listeners;

public interface OnRobotReadyListener {

    /**
     * Called when connection with robot was established.
     *
     * @param isReady {@code true} when connection is open. {@code false} otherwise.
     */
    void onRobotReady(boolean isReady);
}

File: sdk/src/main/java/com/robotemi/sdk/listeners/OnRobotReadyListener.kt
package com.robotemi.sdk.listeners

interface OnRobotReadyListener {

    /**
     * Called when connection with robot was established.
     *
     * @param isReady `true` when connection is open. `false` otherwise.
     */
    fun onRobotReady(isReady: Boolean)
}
